### PR TITLE
add the SQL API to the proxy list

### PR DESCRIPTION
**Pull request by quantumZebraPDX**
[The Dev Studio SQL API](https://developer.domo.com/docs/dev-studio-references/data-api#SQL%20API) doesn't match any of the paths being proxied.

Added sqlQueryPattern to fix this

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class Proxy {
           .then(options => this.transport.request(options))
           .then(rawRequest => rawRequest.pipe(res))
           .catch((err) => {
-            const status = err.statusCode || 500;
+            const status = (err.statusCode !== undefined) ? err.statusCode : 500;
```
Comment (jpumford) [inline src/index.ts]: Here's something that would be a touch more bulletproof:

```suggestion
            const status = (typeof err.statusCode !== 'undefined') ? err.statusCode : 500;
```

**Comment by quantumZebraPDX** [comment]
> Thanks so much for doing this! This looks great, happy to merge and publish it with one quick small change to the error code check

@jpumford I applied your suggestion. Thanks for getting to this fairly quickly!

**Comment by quantumZebraPDX** [comment]
> Awesome! Just pinging @ttingey again to either get Travis set up or force merge this in

@jpumford any update on when you expect this to happen?

**Comment by jpumford** [comment]
@quantumZebraPDX I will ping him again

**Pull request merged**